templates/vars/aks/demo.json

```diff
@@ -3,7 +3,7 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "kubernetesVersion": {
-      "value": "1.19.6"
+      "value": "1.19.7"
```

Comment:
fyi 1.19.9 is out and 1.19.10 should be out soon

Reply:
@timja This PR was to match what we have in Production, we have some lower environments still to update.

We could patch straight to 1.19.9 with this and await next Production rebuild to upgrade to 1.19.x also